# A roadmap towards ubiquitous metagenomic sequencing

Metagenomic sequencing has the potential to [transform clinical diagnosis](https://www.nature.com/articles/s41576-019-0113-7) of infectious disease and [our ability to detect emerging pandemics](https://blogs.scientificamerican.com/observations/how-to-snuff-out-the-next-pandemic/).

![Image](https://lh3.googleusercontent.com/xi-8aqvmLOw4d2EJBbllP57V8OkAmXe_pf4cigggnef8ONqR_lY2qxQY-f1sJkBM-KLgsd2MRp0OBOovjcTH4Eyno6HKXzDGAsMmEDOO3LR7m811_chC06HlIgUQU_ntmLGhYTFvNmQXiCG864QhlA)

- For human respiratory samples, **[sensitivity](https://escherbach.github.io/seq-roadmap/sensitivity) can be comparable to that of qPCR** given sufficient sequencing depth.
- In the context of infectious disease, **time to answer** and **simplicity of sample preparation** is essential. 
- Illumina-style sequencing by synthesis is unlikely to provide sufficient sequencing depth if we are aiming for a sample-to-answer time of ~1 hour. We see two candidate technologies that could provide sufficient sequencing depth in this timeframe at a cost comparable with qPCR. These are biological nanopores and single-molecule optical sequencing. See discussion of [platforms](https://escherbach.github.io/seq-roadmap/platforms).
