#### ## Individual monitoring 

In an individual monitoring approach, samples would be isolated from individuals for testing. This could be done either at the point of care or in a sentinel context. Below, we focus on the former.

In samples acquired from patients infected by SARS-CoV-2, the total RNA from nasopharyngeal samples contains viral RNA at up to 5% when ribosomal RNA (rRNA) is depleted.[⁵](https://www.zotero.org/google-docs/?OU3DwX) At this level of abundance, direct sequencing of total RNA would almost certainly reveal the presence of a novel, SARS-CoV-2-like respiratory virus.[^1]

In addition to this abundance of signals for pathogen detection, individuals showing symptoms (fever and or flu-like symptoms) may already be inclined to seek medical assistance. Reports suggest that patients increasingly want to receive a diagnosis rather than “non-specific upper respiratory infection”, the majority diagnosis in ambulatory care visits for respiratory issues.

For instance, Emily Volk[⁶](https://www.zotero.org/google-docs/?qDxcPN) reports:

> *“Frankly, patient experience scores increase when patients leave our care and have a name for the thing that is bothering them even if there isn’t a targeted antibiotic or antimicrobial that has been identified, that would work against the pathogen. It still is comforting to patients and their families to have a name for the illness that they have.”*

Clinicians themselves are becoming more inclined to order broader panels. According to Emmanuel André[⁶](https://www.zotero.org/google-docs/?DXh03H):

> *“The single pathogenic approach will progressively disappear. Why? Because it’s simply not efficient from a clinical perspective, from a lab perspective and care requires a more comprehensive approach. Where I think we’re going is toward broader panels.”*

Metagenomic sequencing provides the broadest of all possible panels. As such, a metagenomic sequencing-based approach, if it can be as cheap as qPCR testing, may quickly gain widespread adoption even in the absence of planned pandemic prevention efforts.

Any unknown pathogen sequence identified by these tests may be transmitted to pandemic monitoring centers, allowing novel human pathogen cases to be identified almost as soon as they occur.

## Environmental monitoring 

In an environmental monitoring approach, data is collected on pathogen genomic material shed into the environment. Example samples include wastewater or air filters at key locations. Pathogen-targeted environmental monitoring approaches have successfully been used during the COVID-19 pandemic to monitor variants circulating within a population[³](https://www.zotero.org/google-docs/?WbaZB1).

Metagenomics-based environmental monitoring approaches have the potential to perform pathogen-agnostic monitoring at the population level[⁴](https://www.zotero.org/google-docs/?ivdYrO), but face distinct challenges in system sensitivity and cost. Additionally, they lack a commercial engine to drive their ubiquitous adoption.

However, metagenomics-based environmental monitoring may eventually be worth the cost in a layered biodefense system (i.e. for detecting pathogens with a longer latency or subtle initial symptoms), especially if progress in high-throughput sequencing, sample preparation (e.g. depletion of known genomic material), and computational analysis is achieved in the coming years. These challenges are worthy of a separate roadmapping exercise. However, in this roadmap, we focus on an approach where we see a clearer path to pathogen-agnostic early outbreak detection within the coming 5-10 years, namely individual monitoring.